themes/openy_themes/openy_carnation/templates/menu/menu--main--primary-menu.html.twig

```diff
@@ -50,12 +50,6 @@
         {% endif %}
       </li>
     {% endfor %}
-     {# Search icon #}
```

Comment:
I do not think we should remove them totally. All other websites who use search still should have Search in their Carnation themes. I think the best way to do this would be extending theme settings, to enable or disable Search from the admin interface.  
Extending theme also should include an upgrade path for those websites which use Search.